### PR TITLE
BaSP-MR-176: private routes

### PR DESCRIPTION
**Pull request by Franro212**
Implementation of private routes.

Fixed styles of footer and aside.

### CODE REVIEW

File: src/routes/superAdmin.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const SuperAdmins = lazy(() => import('Views/SuperAdmin/SAManagement'));
+const FormSuperAdmin = lazy(() => import('Views/SuperAdmin/SAManagement/Form'));
+const Admins = lazy(() => import('Views/SuperAdmin/AdminManagement'));
+const AdminForm = lazy(() => import('Views/SuperAdmin/AdminManagement/Form'));
+
+const SuperAdminsRoutes = () => {
+  return (
+    <Switch>
+      <Route exact path="/superAdmins/admins" component={Admins} />
+      <Route exact path="/superAdmin/admin/form" component={AdminForm} />
```
Comment (Eugeniagassman) [inline src/routes/superAdmin.js]: this should be superAdmins/admins in plural

Comment (Franro212) [reply src/routes/superAdmin.js]: done

**Comment by LucianoStradiot** [comment]
[[FEATURE] - Private routes](https://trello.com/c/fmT9Ixq0/176-feature-private-routes)

**Review by Eugeniagassman — APPROVED** [review]
very good job!

**Review by LautaroVercellini — APPROVED** [review]
Great job, Fran!

**Pull request merged**